concent_api/core/tests/utils.py

```diff
@@ -975,11 +979,79 @@ def _get_number_of_eth_block(self):  # pylint: disable=no-self-use
     def _pass_rpc_synchronization(self, _rpc, _address, _tx_sign):  # pylint: disable=no-self-use
         return None
 
-    def claim_deposit_true_mock(self, concent_use_case, requestor_ethereum_address, provider_ethereum_address, subtask_cost):  # pylint: disable=unused-argument, no-self-use
-        return (True, True)
+    def claim_deposit_true_mock(self, subtask_id, concent_use_case, requestor_ethereum_address, provider_ethereum_address, subtask_cost, requestor_public_key, provider_public_key):  # pylint: disable=unused-argument, no-self-use
```

Comment:
I know that we allow more characters in line, but that one should be divide.

Reply:
:+1: done